### DISCUSSION TITLE
RaspAP on Raspberry Pi Zero W: Devices cannot connect to hotspot

### DISCUSSION THREAD

**Discussion by WK10010001100001110110011011001101111** [RaspAP/raspap-webgui#1698] [Q&A]
Hello everyone, 

I have installed RaspAP on a Raspberry Pi Zero W and have connected an additional USB WLAN antenna. Model: AWUS036NH with RT3070 chipset. I connected the Raspberry Pi and the external WLAN antenna to an active USB HUB.

I used these instructions https://raspap.com/#quick and the quick installer. 

The internal and additional WLAN antennas are recognized.

I have set up the hotspot with WLAN1 and can also see the hotspot WLAN. Unfortunately I can't log in.

**My DHCP log looks like this:**

```
Nov 11 17:03:34 dnsmasq[1384]: exiting on receipt of SIGTERM
Nov 11 17:04:50 dnsmasq[1856]: started, version 2.89 cachesize 150
Nov 11 17:04:50 dnsmasq[1856]: compile time options: IPv6 GNU-getopt DBus no-UBus i18n IDN2 DHCP DHCPv6 no-Lua TFTP conntrack ipset nftset auth cryptohash DNSSEC loop-detect inotify dumpfile
Nov 11 17:04:50 dnsmasq[1856]: warning: interface wlan1 does not currently exist
Nov 11 17:04:50 dnsmasq-dhcp[1856]: DHCP, IP range 10.9.141.50 -- 10.9.141.254, lease time 12h
Nov 11 17:04:51 dnsmasq[1856]: no servers found in /etc/resolv.conf, will retry
Nov 11 17:04:51 dnsmasq[1856]: read /etc/hosts - 7 names
Nov 11 17:05:32 dnsmasq[1856]: query[A] one.one.one.one from 192.168.178.53
Nov 11 17:05:32 dnsmasq[1856]: config error is REFUSED (EDE: not ready)
Nov 11 17:05:32 dnsmasq[1856]: query[AAAA] one.one.one.one from 192.168.178.53
Nov 11 17:05:32 dnsmasq[1856]: config error is REFUSED (EDE: not ready)
Nov 11 17:05:32 dnsmasq[1856]: query[A] one.one.one.one from 192.168.178.53
Nov 11 17:05:32 dnsmasq[1856]: config error is REFUSED (EDE: not ready)
Nov 11 17:05:32 dnsmasq[1856]: query[AAAA] one.one.one.one from 192.168.178.53
Nov 11 17:05:32 dnsmasq[1856]: config error is REFUSED (EDE: not ready)
Nov 11 17:10:34 dnsmasq-dhcp[1856]: DHCP packet received on wlan1 which has no address
Nov 11 17:10:35 dnsmasq-dhcp[1856]: DHCP packet received on wlan1 which has no address
Nov 11 17:10:37 dnsmasq-dhcp[1856]: DHCP packet received on wlan1 which has no address
Nov 11 17:10:41 dnsmasq-dhcp[1856]: DHCP packet received on wlan1 which has no address
Nov 11 17:10:48 dnsmasq-dhcp[1856]: DHCP packet received on wlan1 which has no address
Nov 11 17:28:04 dnsmasq[1856]: query[A] one.one.one.one from 192.168.178.53
Nov 11 17:28:04 dnsmasq[1856]: config error is REFUSED (EDE: not ready)
Nov 11 17:28:04 dnsmasq[1856]: query[AAAA] one.one.one.one from 192.168.178.53
Nov 11 17:28:04 dnsmasq[1856]: config error is REFUSED (EDE: not ready)
Nov 11 17:28:04 dnsmasq[1856]: query[A] one.one.one.one from 192.168.178.53
Nov 11 17:28:04 dnsmasq[1856]: config error is REFUSED (EDE: not ready)
Nov 11 17:28:04 dnsmasq[1856]: query[AAAA] one.one.one.one from 192.168.178.53
Nov 11 17:28:04 dnsmasq[1856]: config error is REFUSED (EDE: not ready)
```

<details>
  <summary>More Information</summary>

DHCP WLAN0
![image](https://github.com/user-attachments/assets/04c285e4-0c68-46a8-bc92-81d5eabc3ced)
![image](https://github.com/user-attachments/assets/aefaa6f5-7641-4480-9667-d47d313cbcae)

DHCP WLAN1
![image](https://github.com/user-attachments/assets/5b644b3f-6db6-4550-9200-8ad92a843d49)
![image](https://github.com/user-attachments/assets/dda64bfd-5b84-4cf3-ac0c-4e06e226781e)

Networking:

![image](https://github.com/user-attachments/assets/61b3b448-8e9c-4cb6-8639-78814747617e)
</details>

Can someone please help me. Thank you in advance.

Thanks

**Comment by billz** [comment]
(formatted OP's post for clarity)

The error `warning: interface wlan1 does not currently exist` is your first clue. dnsmasq is reporting "not ready" because the interface is down or not fully initialized.

The AWUS036NH adapter has in-kernel drivers and is a good choice, however USB hubs and add-on hardware are known to create under voltage issues with the Zero models. `dmesg | grep undervoltage` may indicate this.

Low power has possibly also corrupted your SD card. Get another one, check your power supply, flash a new card, eliminate the hub and try again. Alternatively, test your setup with a Pi 3b/4.
